### ISSUE TITLE
Contact form bugs

### ISSUE THREAD

**Issue by a-kosanovic** [ALASdoo/alasdoo_com_v2#189]
- [x] `aria-invalid` remains after the field has been filled 
- [x] show error when an invalid file format is uploaded  https://github.com/ALASdoo/alasdoo_com_v2/issues/178
- [x] error message not announced by screen reader 
- [x] remove placeholder text (read out 2x) 
- [x] https://github.com/ALASdoo/alasdoo_com_v2/issues/173

![image](https://github.com/user-attachments/assets/19d21226-0418-42ba-afa7-f4661420d04c)

**Comment by TheKolega** [comment]
Im getting `Broken ARIA reference` error for services-tab, jobs-tab and partnerships-tab (should these reference `#services`, `#jobs` etc, or `#services-tab-pane`, `#jobs-tab-pane` etc

**Comment by TheKolega** [comment]
Looks good :green_circle:

**Comment by Tajana-Somogyi** [comment]
@TheKolega Error messages are inconsistently read with VoiceOver when form is submitted. Sometimes error message for one field from multiple invalid fields is read, sometimes none.
